src/components/paginator.js

```diff
@@ -0,0 +1,79 @@
+const store = new Map()
+
+module.exports = {
+  /**
+   * Create a message with pagination support
+   * @param {String} user - The ID of the user that should be listened to
+   * @param {module:eris.TextChannel} channel - The channel where the message should be sent
+   * @param {Object[]} pages - Array of embed-compatible objects to act as pages for this pagination session
+   * @param {String} text - Optional message to display above all embeds
+   * @returns {Promise<Message>}
+   */
+  init: async (user, channel, pages, text) => {
+    const msg = await channel.createMessage({
+      content: text,
+      embed: pages[0]
+    })
+    await msg.addReaction(String.fromCodePoint(0x23EA)) // rewind
```

Comment:
we should look into using the literal representation of these emotes instead of hexadecimal, like \u23EA. This one is more personal, but it's easier to understand in my eyes

Reply:
Is there a particular reason to use literals or hexadecimal representations in the first place? Seeing as the docs say that [Unicode strings are supported](https://abal.moe/Eris/docs/Message#function-addReaction), wouldn't the most declarative method be to just pass in that instead?